website/docs/concepts/function-components/custom-hooks.mdx

```diff
@@ -5,84 +5,114 @@ description: "Defining your own Hooks "
 
 ## Defining custom Hooks
```

Comment:
I personally am not a fan of this big convoluted example just to attempt to show off custom hooks.
I think we could just show something like custom `use_toggle` hook instead of this complicated thing. (code can even be found here https://github.com/yewstack/yew/discussions/2291#discussioncomment-1861152 )

Reply:
But i don't think this blocks the PR, if you don't want to change it, it will probably be done in the future separately then.

Reply:
I think `use_event` hook is more useful than `use_toggle` so have decided to keep it.